va/va.h

```diff
@@ -1273,6 +1328,32 @@ typedef union _VAConfigAttribValContextPriority{
     uint32_t value;
 }VAConfigAttribValContextPriority;
 
+/** @name Attribute values for VAConfigAttribProtectedContentCipherAlgorithm */
```

Comment:
Same for all these too...

Reply:
I see 2 different questions - 
1. Why are these changes up-streamed?
There is a little bit history behind these changes. We originally designed LibVA Protected APIs in va_protected_content.h header with the intention that this header will be included by a video player that may not be open source or up-streamed. Since Chrome video player is open source/up-streamed and this player includes certain definitions from va_protected_content header so we thought it would be better to have these definitions up-streamed.  
2. Why are these changes moved from va_protected_content.h to va.h?
Originally, LibVA Protected APIs were designed in modularized fashion so that these APIs could potentially be removed in case of regression or lack of open source/up-streamed implementation but with Chrome this situation has changed so we moved some of va_protected_content definitions in va.h